manifests/base/dex/argocd-dex-server-deployment.yaml

```diff
@@ -39,3 +39,18 @@ spec:
       volumes:
       - emptyDir: {}
         name: static-files
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/name: argocd-dex-server
+              topologyKey: kubernetes.io/hostname
+          - weight: 5
+            podAffinityTerm:
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/part-of: argocd
+              topologyKey: kubernetes.io/hostname
```

Comment:
I'm pretty sure the way we run dex we cannot scale it. So I think this should be removed as to not give false impression.

Reply:
removed